src/c/admin-notification-history.js

```diff
@@ -0,0 +1,34 @@
+window.c.AdminNotificationHistory = ((m, h, _, models) => {
+    return {
+        controller: (args) => {
+            const notifications = m.prop([]),
+                getNotifications = (user) => {
+                    let notification = models.notification;
+                    notification.getPageWithToken(m.postgrest.filtersVM({user_id: 'eq', sent_at: 'is.null'}).user_id(user.id).sent_at(!null).order({sent_at: 'desc'}).parameters()).then(function(data){
+                        notifications(data);
+                    });
+                    return notifications();
```

Comment:
Guess we don't need to return here. We can actually use notifications as the callback to the promise: `.then(notifications)`